hermes/file_gen.go

```diff
@@ -10,21 +23,26 @@ type HermesFile struct {
 	contents string
 }
 
+// method of HermesFile generates HermesFile.name of the form Hermes_id_checksum where id is an integer & id <= 50
 func (file *HermesFile) generateFileName(file_id int, file_checksum string) {
-	file.name :=  fmt.Sprintf("Hermes_%02d_%v", file_id, file_checksum)
+	file.name = file.name :=  fmt.Sprintf("Hermes_%02d_%v", file_id, file_checksum)
 }
 
-func (file *HermesFile) generateFileContents(file_ID int) {
+// method of HermesFile generates HermesFile.contents now a string without any significance in the future a pseudo random byte generator will be used
+func (file *HermesFile) generateFileContents(file_id int) {
 	file.contents = "jhfvjhdfjhfjjhjhdfvjvcvfjh";
 }
+
+// method of HermesFile generates the checksum of the file contents
 func (file HermesFile) generateFileChecksum() string {
-	
 	file_contents := []byte(file.contents)
 	hash := sha1.Sum(file_contents)
-	checksum := fmt.Sprintf("%s%x","_", hash)
-	return checksum;
+	// return checksum in hex notation
```

Comment:
We don't need this comment  - in general you should use comments to explain why you do something, now what you're doing.

Reply:
@deniadelcid suggested this comment